ban/__init__.py

```diff
@@ -73,6 +72,7 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
             mapped_results = []
             for result in resp_json:
                 mapped_result = {}
+                mapped_result['MailCityStateZip'] = f"{result.get('mailcity')} {result.get('mailstate')} {result.get('mailzip')}" #pylint: disable=line-too-long
```

Comment:
should we add trim on this in case of empty values?

Reply:
https://github.com/SFDigitalServices/business-locations-fn-py/pull/4/commits/83c7896dc357e988490216197f490805c2f9cc16